sdk/python/feast/client.py

```diff
@@ -682,6 +689,8 @@ def get_online_features(
         """
 
         try:
+            if entity_rows and isinstance(entity_rows[0], dict):
```

Comment:
We should have a deprecation warning for the old format I think

Reply:
Not sure if we want to deprecate Entity Rows entirely as I don't we have a native Python equivalent of corner types like `int32` or `float`. Entity Rows can act like a cop out when users want to specify type information.